src/frontend/src/common/redux/actions/index.ts

```diff
@@ -9,3 +9,18 @@ export const setPathogen: ActionType<Pathogen> = (pathogen) => ({
   type: CZGEReduxActions.SET_PATHOGEN_ACTION_TYPE,
   payload: pathogen,
 });
+
+export const addNotification: ActionType<Partial<ReduxNotification>> = (notification) => ({
```

Comment:
Won't `Partial` here mean that all the keys in `ReduxNotification` are optional? Looking at that type, some of them are explicitly marked as optional, so it doesn't seem right to then implicitly mark all of them as optional when making the action. Right now wouldn't it happily accept not getting a `reduxId` which is required downstream by reducer and displayed components?